PC02_graffiti_BRAATEN.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  4 15:09:16 2020
+
+@author: conno
+"""
+from turtle import *
+
+
+panel = Screen()
+w=750
+h=750
+panel.setup(width=w, height=h)
+
+Turtle()
+from turtle import *
+
+image = "Bezos.gif"
+panel.bgcolor("#ffa8cb")
+panel.bgpic(image)
+
+panel = Screen()
+
+t =Turtle()

```

Comment:
Add a space for better readability
```suggestion
t = Turtle()

```